test/python/compiler/test_transpiler.py

```diff
@@ -1465,32 +1465,19 @@ def test_target_ideal_gates(self, opt_level):
         theta = Parameter("θ")
         phi = Parameter("ϕ")
         lam = Parameter("λ")
-        target = Target(num_qubits=2)
-        target.add_instruction(UGate(theta, phi, lam))
-        target.add_instruction(CXGate())
-        target.add_instruction(Measure())
+        target = Target(2)
```

Comment:
This makes the description field of the target `2` was that your intent?

Reply:
that was unintentional, reverted.